.github/actions/publish-nuget/New-NuGetPackage.ps1

```diff
@@ -16,7 +16,16 @@
 
 param([array] $Arguments)
 
-foreach ($project in (dotnet sln list | Select-Object -Skip 2 | Get-Item))
+if (!(Get-ChildItem *.sln))
```

Comment:
Let's stick with `-not` for neqation instead of the undocumented exclamation mark. However if you swap the if-else then you don't need any negation in the first place and save an extra pair of parens too.

Also `Test-Path` would be more expressive here than `Get-ChildItem`.

Reply:
Isn't the first one something the analyzers should catch?

Reply:
I agree, it should. [I've opened an issue.](https://github.com/PowerShell/PSScriptAnalyzer/issues/1826)

Reply:
Thank you.